File: 1009-complement-of-base-10-integer/1009-complement-of-base-10-integer.py
class Solution:
    def bitwiseComplement(self, n: int) -> int:
        if n==0:
            return 1
        counter = 0
        exp = 0
        while n>0:
            rem = n%2
            if rem==0:
                counter += 2**exp
            n = n//2
            exp+=1
        return counter
        

File: 1009-complement-of-base-10-integer/1009-complement-of-base-10-integer.cpp
class Solution {
public:
    int bitwiseComplement(int n) {
        if(n==0) return 1;
        int counter =0;
        int exp=0;
        while(n>0){
            int rem = n%2;
            n/=2;
            if(rem==0){
                counter+= pow(2,exp);
            }
            exp++;
        }
        return counter;
    }
};